model-optimizer/mo/utils/summarize_graph.py

```diff
@@ -71,9 +71,10 @@ def summarize_graph(graph_def):
         print("[ ERROR ] Both keys were provided --input_model and --input_dir. Please, provide only one of them")
         sys.exit(1)
     tags = argv.saved_model_tags.split(",")
-    graph_def, _, _ = load_tf_graph_def(graph_file_name=argv.input_model, is_binary=not argv.text,
-                                        checkpoint=argv.input_checkpoint,
-                                        model_dir=argv.saved_model_dir, saved_model_tags=tags)
+    graph_def = load_tf_graph_def(graph_file_name=argv.input_model, is_binary=not argv.text,
```

Comment:
I think we need to fix `load_tf_graph_def` inside. We need to align its behavior for all cases so that it always returns a triple. Now it varies that is wrong.

Reply:
This function already returns triple for all cases (in the master branch).

Reply:
Please reproduce the issue and investigate in what cases it does not return triple. Otherwise, I see no issue here and this fix is not needed